wemake_python_styleguide/visitors/ast/annotations.py

```diff
@@ -17,6 +17,84 @@
 from wemake_python_styleguide.visitors.decorators import alias
 
 
+@final
+@alias('visit_any_function', (
+    'visit_FunctionDef',
+    'visit_AsyncFunctionDef',
+))
+class WrongNestedAnnotationVisitor(BaseNodeVisitor):
+    """Ensures that nested annotations are used correctly."""
+
+    _flat_types = {'Literal', 'Union', 'Annotated'}
```

Comment:
We use `ClassVar[FrozenSet[str]]` annotation for this cases and `frozenset()`

Reply:
Done